### PR TITLE
Authentication- has both signup and login screens

### PR DESCRIPTION
**Pull request by Artistole**
Sign up
https://8079ox.csb.app/
image

Login 
https://wf4ty2.csb.app/
image

### CODE REVIEW

File: Login/style.css [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+}
+
+.serach-bar {
+	position: relative;
+	left: 10rem;
+	height: 2rem;
+	width: 35rem;
+	border: 2px solid var(--primary-color);
+	padding: 1.5rem;
+	padding-left: 3rem;
```
Comment (Ayushi20-19) [inline Login/style.css:69]: u can use shorthand for this

Comment (Ayushi20-19) [inline Login/style.css:69]: u can use shorthand for this

**Review by Ayushi20-19 — APPROVED** [review]
suggestion: try using shorthands

**Comment by AnkurChunekar** [comment]
Hey Naga, great work, 
I think you haven't included the confirm password and Name input field, please check LMS, it is mentioned there to have 2 fields for password and an input for user to enter name.

**Comment by Artistole** [comment]
> suggestion: try using shorthands

yeah i have made all of  the changes you have pointed out

**Comment by Artistole** [comment]
> Hey Naga, great work, I think you haven't included the confirm password and Name input field, please check LMS, it is mentioned there to have 2 fields for password and an input for user to enter name.

yeah i have made all of the changes you have pointed out

**Pull request merged**